broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -296,32 +569,32 @@ class ServiceFabrikApiController extends FabrikBaseController {
       .value();
     return Promise
       .try(() => {
-        if (!backup_guid && !time_stamp) {
-          throw new BadRequest('Invalid input as backup_guid or time_stamp not present');
-        } else if (backup_guid) {
-          return this.validateUuid(backup_guid, 'Backup GUID');
-        } else if (time_stamp) {
-          return this.validateDateString(time_stamp);
+        if (!backupGuid && !timeStamp) {
+          throw new BadRequest('Invalid input as backupGuid or timeStamp not present');
+        } else if (backupGuid) {
+          return this.validateUuid(backupGuid, 'Backup GUID');
+        } else if (timeStamp) {
+          return this.validateDateString(timeStamp);
         }
       })
       .then(() => this.backupStore
-        .getBackupFile(time_stamp ? {
-          time_stamp: time_stamp,
-          tenant_id: tenant_id,
-          instance_id: instance_id,
-          service_id: service_id
+        .getBackupFile(timeStamp ? {
```

Comment:
This really does not help readability in my opinion. Better to have an explicit condition check

Reply:
Minor comment: Will this be taken care of?